qless/__init__.py

```diff
@@ -129,6 +130,8 @@ def __init__(self, url='redis://localhost:6379', hostname=None, **kwargs):
         import socket
         # This is our unique idenitifier as a worker
         self.worker_name = hostname or socket.gethostname()
+        if sys.version_info[0] >= 3:
+            kwargs['decode_responses'] = True
```

Comment:
I'm not sure if we guarantee that all job data is valid Unicode currently. This could potentially cause errors if that assumption doesn't hold.

But I do like the simplification that this makes in porting to Python 3.